runtime/kusama/src/lib.rs

```diff
@@ -454,13 +454,15 @@ impl pallet_collective::Trait<CouncilCollective> for Runtime {
 
 parameter_types! {
 	pub const CandidacyBond: Balance = 1 * DOLLARS;
-	pub const VotingBond: Balance = 5 * CENTS;
+	pub const VotingBondBase: Balance = 5 * DOLLARS;
```

Comment:
I think a 100x increase here is too much? I think `1 DOLLAR` is a good middle ground.